src/bramble/client/main.js

```diff
@@ -122,7 +122,34 @@ define([
 
     // Expose Filer for Path, Buffer, providers, etc.
     Bramble.Filer = Filer;
-    var _fs = new Filer.FileSystem();
+    var _fs;
+
+    // Set URL param for memory-backed FileSystem
+    var mem_filesystem = "BrambleMemoryFileSystem";
+
+    // Splits the parameter from the URL
+    function getUrlParams() {
+      var param = decodeURIComponent( window.location.href.slice( window.location.href.indexOf( '?' ) + 1 ) );
+      return param;
+    }
+
+    // Check if the parameter is equal to 'BrambleMemoryFileSystem'
+    if(getUrlParams() === mem_filesystem) {
+      console.log("Using memory-backed filesystem");
```

Comment:
Change this comment to `[Bramble] Overriding filesystem to use temporary storage. ALL FILES WILL BE DELETED WHEN PAGE IS CLOSED.`

Reply:
nit: change this to `console.warn` vs. `console.log`